README.md

```diff
@@ -129,6 +129,42 @@ optimization:
 ```
 
 ## Custom Dataset
+# Feature extraction & (pre-trained) backbones
```

Comment:
The formatting is a bit off here. I think this header
`## Custom Dataset`
should be placed under your section, and
`# Feature extraction & (pre-trained) backbones`
should be 
`## Feature extraction & (pre-trained) backbones`